### ISSUE TITLE
Contrast ratio too low for placeholder text in input fields in dark mode.

### ISSUE THREAD

**Issue by niik** [desktop/desktop#5978]
#5968 raised the issue that while using the dark theme the placeholder text in input fields doesn't meet the WCAG AA minimum contrast ratio.

<img width="263" alt="screen shot 2018-10-23 at 16 53 35" src="https://user-images.githubusercontent.com/634063/47369602-531b8480-d6e4-11e8-8ae5-458c91aaafb5.png">

<img width="715" alt="screen shot 2018-10-23 at 16 55 20" src="https://user-images.githubusercontent.com/634063/47369658-77776100-d6e4-11e8-81ab-655a242d4cae.png">

We should bump the contrast a bit here to get us up to 4.5:1. I don't believe we currently do any specific styling of placeholder text so this is likely a user agent derived color.

cc @desktop/design
